packages/muelu/adapters/xpetra/MueLu_RefMaxwell_decl.hpp

```diff
@@ -76,7 +76,7 @@
 #include "Xpetra_CrsMatrixWrap_fwd.hpp"
 #include "Xpetra_ExportFactory_fwd.hpp"
 
-#ifdef HAVE_MUELU_IFPACK2
+#if defined(HAVE_MUELU_IFPACK2) && (!defined(HAVE_MUELU_EPETRA) || defined(HAVE_MUELU_TPETRA_INST_INT_INT))
```

Comment:
I don't understand why the second condition is required.

Reply:
I mean the case (If Ifpack2  and not Epetra).

Reply:
We need Ifpack2 for Hiptmair smoothing. Also either we want no Epetra, in which case the Tpetra::MultiVector below is defined, or, if we have Epetra, then we want the corresponding Tpetra instantiation to be available.